evansmusomi/burger-builder
# Uses Redux for Order Actions

*What this does*
- Adds order actions
- Adds order reducer
- Connects order reducer to the checkout process
- Reconfigures the burger builder to read the relevant redux store slice

*How to verify*
- A user can build a burger and proceed to checkout successfully
- A developer can use redux dev tools to verify order actions are dispatched
- A developer can use redux dev tools to verify the order and burgerBuilder store slices exist
